### DISCUSSION TITLE
Differences Between Windows Builds 5.15.2 and 6.3.1?

### DISCUSSION THREAD

**Discussion by loyukfai** [xiaoyifang/goldendict-ng#187] [General]
Just asking...

<img width="899" alt="image" src="https://user-images.githubusercontent.com/23282935/199386725-d344fa6b-de41-44e3-9f74-0b391e168f84.png">

**Comment by shenlebantongying** [comment]
5.15.2 and 6.3.1 are Qt versions. They are supposed to be "identical".

6.xx might have some bugs because it is very new but I think 5.xx will be dropped someday.

**Reply by loyukfai** [reply]
Thanks!

BTW, Qt 6 was released 2 years ago, still not mature for daily use?

Cheers.

**Reply by xiaoyifang** [reply]
> Qt 6 was released 2 years ago, still not mature for daily use?

sadly it is true. with each Qt6 minor version,it introduce new features while break some other features.
From the point of goldendict daily usage.

**Reply by loyukfai** [reply]
Thanks! Will stick with 5.xx for now then.

**Reply by xiaoyifang** [reply]
btw,qt6 do offer speed compared to qt5.15 and fixed some annoying bugs.
